hal/src/common/gpio/v2.rs

```diff
@@ -1,19 +1,19 @@
 //! # Version 2 of the GPIO module
 //!
-//! The new API provides two different submodules, [pin] and [dynpin],
-//! representing two different ways to handle GPIO pins. The default, [pin], is
-//! a type-level API that tracks the state of each pin at compile-time. The
-//! alternative, [dynpin] is a type-erased, value-level API that tracks the
+//! The new API provides two different submodules, [`pin`] and [`dynpin`],
```

Comment:
'New ' is relative, maybe call this 'the GPIO API' ?

Reply:
How about

> The `v2` API provides two different submodules...

Does that work?